Update Office365ProPlus.nuspec

hanthor opened the pull request:
Adding a link to Microsoft's tool for creating a custom config.xml

hanthor committed:
Update Office365ProPlus.nuspec

Adding a link to Microsoft;s tool for creating a custom config.xml

mikecole commented on manual/office365proplus/Office365ProPlus.nuspec:
```diff
@@ -13,14 +13,16 @@
 subscription. This will download and silently install the entire Office 365 2016 Professional Plus edition. A license is required
 and you will be prompted to enter your account information upon first launching the software after install.
+      
+You can creat a custom config.xml at https://config.office.com/deploymentsettings
```
typo: 'creat'

mikecole commented:
Thanks for the PR! I marked one typo. If you fix that, plus increment the second segment of the "version" to today's date, I'll go ahead and push. Thanks again!

hanthor committed:
Update Office365ProPlus.nuspec

Fixed typo and incremented version number. Should the version be 2019 now? Since office 2019 is out

Office 365 uses the four digit naming convention. So 1903 would be the current version

mikecole commented:
@hanthor  I just installed this package on a VM and did not see any references to 2019. Is there something I'm missing? I've always been a little confused about Office 365 product versioning. If you feel it should be 2019 let me know why.

hanthor commented:
https://docs.microsoft.com/en-us/officeupdates/update-history-office365-proplus-by-date

mikecole commented:
Good enough for me. If you want to adjust that in your PR I'll push it.
Thanks!

On Mon, Apr 8, 2019 at 2:50 PM hanthor <notifications@github.com> wrote:

>
> https://docs.microsoft.com/en-us/officeupdates/update-history-office365-proplus-by-date
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/mikecole/chocolatey-packages/pull/55#issuecomment-480980860>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABHL6-mf6vISgewO3bnusaWMdWQz83UOks5ve517gaJpZM4cifhA>
> .
>

hanthor committed:
Update Office365ProPlus.nuspec

mikecole commented on manual/office365proplus/Office365ProPlus.nuspec:
```diff
@@ -3,7 +3,7 @@
   <metadata>
     <id>office365proplus</id>
-    <version>2016.20190408</version>
+    <version>1903.20190408</version>
```
choco will not recognize this as a newer package.

hanthor commented:
@mikecole shoot I’ll just change it back to 2019?

On Mon, Apr 8, 2019 at 3:55 PM Mike Cole <notifications@github.com> wrote:

> *@mikecole* commented on this pull request.
> ------------------------------
>
> In manual/office365proplus/Office365ProPlus.nuspec
> <https://github.com/mikecole/chocolatey-packages/pull/55#discussion_r273212335>
> :
>
> > @@ -3,7 +3,7 @@
>  <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
>    <metadata>
>      <id>office365proplus</id>
> -    <version>2016.20190408</version>
> +    <version>1903.20190408</version>
>
> choco will not recognize this as a newer package.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mikecole/chocolatey-packages/pull/55#pullrequestreview-224053472>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFkeORd6fp58rLzPWUauHvnwStHLbDgVks5ve56SgaJpZM4cifhA>
> .
>

mikecole commented:
I think that's probably best for now. I'm not sure why I changed the
versioning to 2016.* but it seems to have not been the right decision.

On Mon, Apr 8, 2019 at 4:11 PM hanthor <notifications@github.com> wrote:

> @mikecole shoot I’ll just change it back to 2019?
>
> On Mon, Apr 8, 2019 at 3:55 PM Mike Cole <notifications@github.com> wrote:
>
> > *@mikecole* commented on this pull request.
> > ------------------------------
> >
> > In manual/office365proplus/Office365ProPlus.nuspec
> > <
> https://github.com/mikecole/chocolatey-packages/pull/55#discussion_r273212335
> >
> > :
> >
> > > @@ -3,7 +3,7 @@
> > <package xmlns="
> http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
> > <metadata>
> > <id>office365proplus</id>
> > - <version>2016.20190408</version>
> > + <version>1903.20190408</version>
> >
> > choco will not recognize this as a newer package.
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/mikecole/chocolatey-packages/pull/55#pullrequestreview-224053472
> >,
> > or mute the thread
> > <
> https://github.com/notifications/unsubscribe-auth/AFkeORd6fp58rLzPWUauHvnwStHLbDgVks5ve56SgaJpZM4cifhA
> >
> > .
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mikecole/chocolatey-packages/pull/55#issuecomment-481009624>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABHL69i3xM0QM6BsInrzgVRflybj5QJiks5ve7BkgaJpZM4cifhA>
> .
>
